### DISCUSSION TITLE
Container Block

### DISCUSSION THREAD

**Discussion by sloyer** [WebDevStudios/nextjs-wordpress-starter#413] [Ideas]
With the new ACF, it would be possible to support container blocks using innerblocks. 

It would be awesome to have a container Block where users could select width constraints and backgrounds and nest other divs inside of it. 

1. I select a Container Block
2. I select width option 100%, 960px, etc
3. I can select background color or image to show on Div
4. I can nest other containers or blocks inside of it. 

Use Case: 
I want to select a background image and have it full width. But inside I want the content to be constricted in another container to 960px.

**Comment by sloyer** [comment]
It looks like the column block already pretty much does this, going to try and see if I can use whats there with ACF to do this =)

**Comment by sloyer** [comment]
Gave this a shot, I can get the block, place things inside the block, get the top level of the block (containing div) Then it will render the divs but none of the content inside of it.  This is heavily based off the Column Block as i figured since it does essentially the same thing it should work for this case. 

Any insight to this would be awesome as this would be a very valuable block i think to have. 


[ContentContainer.zip](https://github.com/WebDevStudios/nextjs-wordpress-starter/files/6423942/ContentContainer.zip)

**Reply by yoryi** [reply]
Well, I think you need to create Columns block for your purpose

**Comment by sloyer** [comment]
So it appears blocks from ACF generate a bit different of a structure for container blockers using <innerBlocks />

Column block rendering innerblocks
<img width="875" alt="Screen Shot 2021-05-05 at 8 01 45 PM" src="https://user-images.githubusercontent.com/1777605/117240292-f634d500-addc-11eb-9c4c-9545f2812c4f.png">

My ACF block rendering innerBlocks
<img width="845" alt="Screen Shot 2021-05-05 at 8 02 07 PM" src="https://user-images.githubusercontent.com/1777605/117240314-0056d380-addd-11eb-9d84-45f26bee9e7c.png">

So just need to figure out a way to utilize this
